module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -301,15 +301,18 @@ public function supportsConnectingLibraryCard($authMethod = null)
     }
 
     /**
-     * Is persistent login supported?
+     * Is persistent login supported by the authentication method?
+     *
+     * @param string $method Authentication method (overrides currently selected method)
      *
      * @return bool
      */
-    public function supportsPersistentLogin()
+    public function supportsPersistentLogin(?string $method = null): bool
     {
         if (!empty($this->config->Authentication->persistent_login)) {
-            $method = $this->getAuth() instanceof ChoiceAuth
-                ? $this->getAuth()->getSelectedAuthOption() : $this->getAuthMethod();
+            if (null === $method) {
+                $method = $this->getSelectedAuthMethod();
```

Comment:
Could we just do `$method ?? $this->getSelectedAuthMethod()` below to simplify this?

Reply:
Good idea. Done!